src/API/Exception/ZoneSettingFailException.php

```diff
@@ -0,0 +1,6 @@
+<?php
+namespace CF\API\Exception;
+class ZoneSettingFailException extends CloudFlareException
+{
+    protected $message = 'Oops, something went wrong, please try again in a few minutes';
```

Comment:
Agreed we should, but don't have the bandwidth to work on it right now.  Its probably needs to wait until we redesign one of the other plugins.